app/styles/application.scss

```diff
@@ -1,3 +1,6 @@
+---
+eleventyExcludeFromCollections: true
+---
```

Comment:
Ah, now remembering the change I made elsewhere. For the CORE Design History, I changed it so that homepage showed items from `collections.post` as opposed to `collections.all`: https://github.com/communitiesuk/core-design-history/commit/be858cb68907ca026ff833f1553c749c1f869780#diff-35ba1fcc4dd53a830c4cf6df0b4191b0c578176041e6b5e757abdf5cd64d1907

This matches with the change we made to `index.md` in this project, where we also loop through `collections.post`.

I think the issue may lie in the homepage template: https://github.com/x-govuk/govuk-design-history/blob/d7da0985133b2dbdce2360d5904e873b3fc380df/app/_layouts/home.njk#L38

Maybe the better fix would be to make the following change:

```diff
- {% for item in collections.all | eleventyNavigation(options.homeKey) %}
+ {% for item in collections.post | eleventyNavigation(options.homeKey) %}
```

(Eleventy doesn’t provide native support for CSS processing, so the plugin adds support for Sass, processing any files that end in `.scss`, as documented here https://www.11ty.dev/docs/languages/custom/. As we’re processing these files, they will appear in `collections.all`.)

Reply:
I wonder whether, rather than tagging everything with “post” or the name of the section/service, we could use the eleventyNavigation plugin to iterate through all the posts instead?

That’d also start to let us use tags for other purposes such as ad-hoc collections of related posts?